cds/modules/theme/templates/cds_theme/scripts.html

```diff
@@ -1,15 +1,3 @@
 {# -*- coding: utf-8 -*- -#}
 
 {% assets "cds_theme_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
-
-<script type="text/x-mathjax-config">
-  MathJax.Hub.Config({
-    tex2jax: {
-      inlineMath: [['$','$']],
-      processEscapes: true
-    },
-    showProcessingMessages: false,
-    messageStyle: "none"
-  });
-</script>
-<script type="text/javascript" src="//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
Can you put this part in a different commit? ☺️

Reply:
You are perfectly right, done!